internal/namespaces/autocomplete/autocomplete_test.go

```diff
@@ -3,9 +3,91 @@ package autocomplete
 import (
 	"testing"
 
+	"github.com/scaleway/scaleway-cli/internal/core"
 	"github.com/stretchr/testify/assert"
 )
 
+func Test_Autocomplete(t *testing.T) {
+	t.Run("zsh", func(t *testing.T) {
+		t.Run("Level1", core.Test(&core.TestConfig{
+			Commands: GetCommands(),
+			Cmd:      "scw autocomplete complete zsh -- 1",
+			Check: core.TestCheckCombine(
+				core.TestCheckGolden(),
```

Comment:
golden files should be used to test UI, not to test the content of a result 

can we have the response inlined here please ?

Reply:
also, don't forget to remove the unused golden files